packages/device_info_plus/device_info_plus/example/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>9.0</string>
+  <string>11.0</string>
```

Comment:
This happens automatically when you are trying to run the example app with latest Flutter and aligns with bump of minimum supported iOS version to 11.